### PR TITLE
refactor(services/tasks): Swtich to dify_config witch Pydantic 

### PR DESCRIPTION
**Pull request by ox01024**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue #5937 
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by laipz8200** [comment]
Hi there! Before submitting any changes, it's a good idea to leave a comment in the issue and confirm the scope of your modifications. Otherwise, your code might not get merged.

At the moment, your changes haven't conflicted with anyone else's work. Would you like me to help you mark the rag, service, and tasks directories?

**Comment by ox01024** [comment]
I've incorporated the scope of modifications where I deemed fit, including refactoring services and tasks. (refactor(services/tasks))   However, it now appears inappropriate. Please demonstrate the truly suitable solution.

**Comment by laipz8200** [comment]
I’m sorry for causing any misunderstanding, your work is great, and I hope you can leave a message in #5937 before you start. Since this refactoring project is a community effort, if you don't leave a message, I might assign these directories to someone else while you're working on them.

For now, I'll assign "services" and "tasks" to you and prepare to merge it.

File: api/services/app_generate_service.py [hunk 1]
```diff
@@ -89,8 +89,8 @@ def generate(cls, app_model: App,
     def _get_max_active_requests(app_model: App) -> int:
         max_active_requests = app_model.max_active_requests
         if app_model.max_active_requests is None:
-            from flask import current_app
-            max_active_requests = int(current_app.config['APP_MAX_ACTIVE_REQUESTS'])
+            from configs import dify_config
```
Comment (laipz8200) [inline api/services/app_generate_service.py]: This line can be moved to the top of the file.

**Review by laipz8200 — CHANGES_REQUESTED** [review]
LGTM. Please run `dev/reformat` to resort imports and ensure the CI check is successful.

**Review by laipz8200 — APPROVED** [review]

**Pull request merged**